packages/editor/src/utils/index.js

```diff
@@ -6,3 +6,4 @@ import mediaUpload from './media-upload';
 export { mediaUpload };
 export { cleanForSlug } from './url.js';
 export { getTemplatePartIcon } from './get-template-part-icon';
+export { getItemTitle } from './get-item-title';
```

Comment:
This import statement means to make the `getItemTitle()` function public. However, since this function is only to be used within the editor package, there is no need to make it public.

Reply:
Oh yeah, good spot! Addressed in https://github.com/WordPress/gutenberg/pull/67650/commits/30a7f2acb1cd2c784655f3436ce2b081dcaafd41.